doc/source/conf.py

```diff
@@ -128,12 +128,21 @@
 
 
 # -- Options for HTML output -------------------------------------------------
+cname = os.getenv("DOCUMENTATION_CNAME", "<DEFAULT_CNAME>")
+"""The canonical name of the webpage hosting the documentation."""
 html_theme = "ansys_sphinx_theme"
 html_logo = pyansys_logo_black
 html_theme_options = {
     "github_url": "https://github.com/pyansys/grantami-bomanalytics",
-    "additional_breadcrumbs": [("PyAnsys Documentation", "https://docs.pyansys.com/")],
+    "additional_breadcrumbs": [
+        ("PyAnsys Documentation", "https://docs.pyansys.com/"),
+        ("Granta MI", "https://grantami.docs.pyansys.com/"),
```

Comment:
I'm not sure if publishing will work at all, because the cname is not available.

But it wouldn't affect existing docs, as they are still generated from the other repo

Reply:
At this point we could request bomanalytics.grantami.docs.pyansys.com. For a while we'd have the bomanalytics served both at `bomanalytics.grantami` and `grantami`, but that would enable to sort things through before changing the landing page at `grantami.docs`

Reply:
That makes sense. We could even get to the point of having the bomanalytics docs live, and then only change the grantami page from bomanalytics to the Granta MI page when we want to release recordlists.

Sounds good to me. Yes, go ahead and request bomanalytics.grantami....